node/clients/LMClient.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { InstanceOptions, IOContext } from '@vtex/api'
+import { ExternalClient } from '@vtex/api'
+
+export default class LMClient extends ExternalClient {
+  constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(`http://${ctx.account}.vtexcommercestable.com.br/`, ctx, {
```

Comment:
🟡 This endpoint is public, and the HTTP is not secure ( HTTPS ), so the token is easily intercepted on the internet.
If possible use an internal endpoint or use HTTPS